<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plaMotor</class>
 <widget class="QDialog" name="plaMotor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asynchronous Motor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QComboBox" name="catalogTypeCombo"/>
   </item>
   <item row="9" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="phaseCodeCombo"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="8" column="1">
    <widget class="QTextEdit" name="descriptionTextEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="aliasLineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="uidLineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="connectionTypeLabel">
     <property name="text">
      <string>Connection Type</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="phaseCodeLabel">
     <property name="text">
      <string>Phase Code</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="aliasLabel">
     <property name="text">
      <string>Alias Name</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="catalogTypeLabel">
     <property name="text">
      <string>Catalog Type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="uidLabel">
     <property name="text">
      <string>UID</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="systemIdLabel">
     <property name="text">
      <string>System</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="connectionTypeCombo"/>
   </item>
   <item row="8" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="producerComboBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Producer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="systemComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
